pycheribuild/projects/bluespec_compiler.py

```diff
@@ -45,11 +45,20 @@ def __init__(self, config: CheriConfig):
         self.add_required_system_tool("cabal", apt="cabal-install", homebrew="cabal-install")
         for i in ("autoconf", "gperf", "bison", "flex"):
             self.add_required_system_tool(i, homebrew=i)
+        self.add_required_system_header("tcl/tcl.h", apt="tcl-dev")
         self.make_args.set(PREFIX=self.install_dir)
```

Comment:
If you remove `build_in_source_dir = True`, the following might be sufficient:
```suggestion
        self.make_args.set(PREFIX=self.install_dir, BUILDDIR=self.build_dir)
```

Reply:
Maybe I should just remove the `distclean` workaround and always use `git clean` for in-source clean. For now using this seems fine.

Reply:
I wonder if it works if you set BUILDIR in the environment instead of on the make command line?